src/stores/wpcom-api.ts

```diff
@@ -152,6 +168,14 @@ export const wpcomApi = createApi( {
 			keepUnusedDataFor: 60 * 60,
 			providesTags: [ 'SnapshotUsage' ],
 		} ),
+		getSnapshotStatus: builder.query< LiveSnapshotStatus, number >( {
```

Comment:
```suggestion
		getSnapshotStatus: builder.query< z.infer< typeof snapshotStatusSchema >, number >( {
```
Sorry for the nitpick, but we might as well use `z.infer` here, since we don't reuse `LiveSnapshotStatus` anywhere else.

Reply:
Thanks for the suggestion. I have updated it.